bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableOptionsFactory.java

```diff
@@ -392,16 +392,10 @@ private static void setClientCallOptions(Configuration configuration,
 
     clientCallOptionsBuilder
         .setUseTimeout(configuration.getBoolean(BIGTABLE_USE_TIMEOUTS_KEY, USE_TIMEOUT_DEFAULT));
-    if (configuration.get(BIGTABLE_RPC_TIMEOUT_MS_KEY) != null) {
```

Comment:
Can you please separate out the BigtableOptionsFactory.java changes into a different PR?